vividus-plugin-browserstack/src/main/java/org/vividus/bdd/browserstack/steps/BrowserStackSteps.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.browserstack.steps;
+
+import com.browserstack.automate.model.Session;
+import com.browserstack.client.exception.BrowserStackException;
+import com.google.common.eventbus.EventBus;
+
+import org.jbehave.core.annotations.AfterScenario;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.vividus.browserstack.BrowserStackAutomateClient;
+import org.vividus.reporter.event.LinkPublishEvent;
+import org.vividus.selenium.IWebDriverProvider;
+
+public class BrowserStackSteps
+{
+    private final IWebDriverProvider webDriverProvider;
+    private final EventBus eventBus;
+    private final BrowserStackAutomateClient appAutomateClient;
+
+    public BrowserStackSteps(IWebDriverProvider webDriverProvider, EventBus eventBus,
+            BrowserStackAutomateClient appAutomateClient)
+    {
+        this.webDriverProvider = webDriverProvider;
+        this.eventBus = eventBus;
+        this.appAutomateClient = appAutomateClient;
+    }
+
+    @AfterScenario
+    public void publishReport() throws BrowserStackException
```

Comment:
I believe you can have a sinlge class for all BS steps (only 2)

Reply:
- it's more clear when we separate steps used by users and not by users
- some users may dig into implementation, such steps can bring additional noise

Reply:
if users are skilled enough to read java code, then there should be no problem to have any other methods in the class

Reply:
what is advantage of putting two completely different sets of functionality into one class?